website/docs/release_notes.md

```diff
@@ -7,6 +7,152 @@ import ReactPlayer from 'react-player'
 
 # Release notes
 
+## [v0.2.0](https://github.com/VirtusLab/scala-cli/releases/tag/v0.2.0)
+
+### Require the `--power` option for restricted features by default
+
+Until now, Scala CLI has been limiting some of its functionalities in its `scala` distribution.
+Starting with `v0.2.0`, those limitation will be applied to all distributions, including `scala-cli`.
+Restricted features can be accessed by using the `--power` launcher flag. Do note that launcher flags have to be passed **before** the sub-command.
+```bash ignore
+scala-cli --power package .
+```
+Alternatively, the `power` mode can be turned on globally by running:
+
+```bash ignore
+scala-cli config power true 
+```
+
+Please note that this change may affect your existing scripts or workflows that rely on the limited commands from ScalaCLI (such as `package`, `publish`). You can still use this command with enabled `power` mode.
+
+When you try to use a limited command in restricted mode, you will now see a warning message with suggestions on how to enable this command:
+
+
+```bash ignore
+$ scala-cli package Hello.scala
+# This command is restricted and requires setting the `--power` option to be used.
+# You can pass it explicitly or set it globally by running:
+#    scala-cli config power true
+$ scala-cli config power true
+$ scala-cli package Hello.scala
+# Wrote Hello, run it with
+#   ./Hello
+```
+
+Added by [@lwronski](https://github.com/lwronski) in [#1835](https://github.com/VirtusLab/scala-cli/pull/1835) and [#1849](https://github.com/VirtusLab/scala-cli/pull/1849)
+
+### Allow executable Scala scripts without a file extension
+
+As of this release Scala scripts without the `*.sc` file extension will be supported for execution when using the `shebang` command.
+
+```scala title=hello
+#!/usr/bin/env -S scala-cli shebang -S 3
+
+println(args.size)
+println(args.headOption)
+```
+```bash
+chmod +x hello
+./hello Hello World
+#2
+#Some(Hello)
+ ```
+Note that files with no extension are always run as scripts even though they may contain e.g. a valid `.scala` program.
+
+Also, do note that this feature has only been added for `shebang` - the `run` sub-command (which is the default way of running inputs when a sub-command is not specified explicitly) will not support this.
+
+Added by [@MaciejG604](https://github.com/MaciejG604) in [#1802](https://github.com/VirtusLab/scala-cli/pull/1802)
+
+### Export information about the project in the json format
+
+```bash ignore
+scala-cli --power export --json
+```
```

Comment:
Could we should an example output here? OR part of it?

Reply:
Yeah, so I added basin json output without `Resolvers` field.